src/sidebar/oauth-auth.js

```diff
@@ -37,6 +37,7 @@ function auth($http, settings) {
     return {
       accessToken:  data.access_token,
       expiresIn:    data.expires_in,
+      expiresAt:    new Date(Date.now() + (data.expires_in * 1000)),
```

Comment:
(Previously we got by with an `expiresIn` delta but the new code requires an `expiresAt` time and that time needs to be computed asap after the tokens are received from the server.)